<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>29</y>
     <width>691</width>
     <height>461</height>
    </rect>
   </property>
   <widget class="QWidget" name="login_page">
    <widget class="QWidget" name="widget_11" native="true">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>531</width>
       <height>431</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>241</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 700 40pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Login </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
font: 10pt &quot;Segoe UI&quot;;
border:1px solid white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
font: 10pt &quot;Segoe UI&quot;;
border:1px solid white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Username : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Password :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_login">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>310</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="back2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>380</y>
        <width>172</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_14" native="true">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>200</y>
       <width>241</width>
       <height>611</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/QtWidgetsApplication3/download-removebg-preview.png);
background-position: center;
    background-repeat: no-repeat;
</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="welcome1_page">
    <widget class="QWidget" name="widget_8" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>551</width>
       <height>451</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>411</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Terminal&quot;;</string>
      </property>
      <property name="text">
       <string>WELCOME TO CAIRO RIDE PROGRAM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="create_button">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>133</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
font: 10pt &quot;Segoe UI&quot;;
border:1px solid white;
</string>
      </property>
      <property name="text">
       <string>create a new account</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login_button">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>220</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
font: 10pt &quot;Segoe UI&quot;;
border:1px solid white;
</string>
      </property>
      <property name="text">
       <string>Login to your account</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_9" native="true">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>60</y>
       <width>241</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/QtWidgetsApplication3/download-removebg-preview.png);
background-position: center;
    background-repeat: no-repeat;
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit01">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>640</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>exit program</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="signin_page">
    <widget class="QWidget" name="widget_12" native="true">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>-30</y>
       <width>401</width>
       <height>621</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="lineEdit_username_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
font: 10pt &quot;Segoe UI&quot;;
border:1px solid white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>210</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
font: 10pt &quot;Segoe UI&quot;;
border:1px solid white;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Username : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Password :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>(8 characters or more)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Email :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>(it must have name followed by &quot;@gmail.com&quot;)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_email">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>270</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
font: 10pt &quot;Segoe UI&quot;;
border:1px solid white;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Enter Balance : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_balance">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>350</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
font: 10pt &quot;Segoe UI&quot;;
border:1px solid white;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sign">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>400</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign in</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="back">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>430</y>
        <width>172</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>0</y>
       <width>381</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 700 30pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Create New Account </string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
